File: Week-08/Meherun Nesa Enta/Which Mixture/main.cpp
#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int t;
	cin>>t;
	while(t--)
	{
	    int a,b;
	    cin>>a>>b;
	    if(a==0 && b>0)
	    cout<<"Liquid"<<endl;
	    else if(a>0&& b==0)
	    cout<<"Solid"<<endl;
	    else
	    cout<<"Solution"<<endl;
	}
	return 0;
}

File: Week-08/Meherun Nesa Enta/Which Mixture/main.java
import java.util.*;
import java.lang.*;
import java.io.*;

/* Name of the class has to be "Main" only if the class is public. */
class Codechef
{
	public static void main (String[] args) throws java.lang.Exception
	{
		// your code goes here
			int t;
    Scanner input =new Scanner(System.in);
   t=input.nextInt();
   for(int i=0;i<t;i++)
	{
	    int a,b;
	     a=input.nextInt();
	     b=input.nextInt();
	    if(a==0 && b>0)
	    System.out.println("Liquid");
	    else if(a>0&& b==0)
	     System.out.println("Solid" );
	    else
	     System.out.println("Solution" );
	}
	}
}
